src/lib.rs

```diff
@@ -0,0 +1,8 @@
+pub mod chunk_type;
+pub mod chunk;
+pub mod png;
+pub mod args;
+pub mod error;
+
+pub type Error = Box<dyn std::error::Error>;
+pub type Result<T> = std::result::Result<T, Error>;
```

Comment:
By convention, crate `Error` must be used in crate's `Result`.
I suggest we use:

```rust
pub type Result<T> = std::result::Result<T, error::PngMeError>;
```